packages/client/wallets/aa/src/api/CrossmintWalletService.ts

```diff
@@ -26,9 +26,16 @@ export class CrossmintWalletService extends BaseCrossmintService {
         );
     }
 
-    async getAbstractWalletEntryPointVersion(email: string, chain: EVMBlockchainIncludingTestnet) {
+    async getAbstractWalletEntryPointVersion(
+        userIdentifier: UserIdentifierParams,
+        chain: EVMBlockchainIncludingTestnet
+    ) {
+        const identifier = userIdentifier.email
+            ? `email=${encodeURIComponent(userIdentifier.email)}`
+            : `userId=${userIdentifier.userId}`;
+
         return this.fetchCrossmintAPI(
-            `v1-alpha1/wallets/entry-point-version?email=${encodeURIComponent(email)}&chain=${chain}`,
+            `v1-alpha1/wallets/entry-point-version?${identifier}&chain=${chain}`,
```

Comment:
oh now I see–using query params in GET makes the JSON harder. If possible that would make the code easier in crossbit-main, but if it's too messy then it makes sense to keep as-is